### DISCUSSION TITLE
Incorrect display of map difficulty when playing.

### DISCUSSION THREAD

**Discussion by unqxl** [ppy/osu#24117] [Q&A] [answered]
I just want to make sure, I apologize in advance for possible incorrect wording.

I tested this thing on two modes - osu!standart and osu!mania.
In the skin editor provided the opportunity to display the difficulty of the map.
And the point is that the difficulty of the map is incorrect.

As a first example, let's take osu!mania mode:
- The song selection menu shows 2.74 stars (with 4K mod):
![image](https://github.com/ppy/osu/assets/46648824/c81a520c-86b8-4207-8db2-dbe98f8c3aa0)
- But during the game it displays 5.83 stars.
![image](https://github.com/ppy/osu/assets/46648824/f31432f5-315d-4742-a36d-890ec6157667)

As a second example, let's take osu!standart mode:
- The song selection menu shows 6.96 stars (with HD, DT, Mirror, Difficulty Adjust mods)
![image](https://github.com/ppy/osu/assets/46648824/0743178a-bc83-4fef-8659-212ee169a387)
- But during the game it shows 5.83 stars again.
![image](https://github.com/ppy/osu/assets/46648824/3671e3d1-8649-4592-ae4f-87dc42ca7a1a)

**Comment by bdach** [comment, marked as answer]
duplicate of https://github.com/ppy/osu/issues/21946

**Reply by unqxl** [reply]
Oh, sorry for that 😅
